### ISSUE TITLE
Parental control

### ISSUE THREAD

**Issue by StevenKim523** [AdguardTeam/AdGuardHome#4165]
Parental control doesn't work well. The homepage of porn websites can load normally. The videos on pornhub.com can be even play as usual. And the parental control redirect page disappeared.

**Comment by EugeneOne1** [comment]
@StevenKim523, hello.  Could you please provide some details?

 1. How did you install the AGH?
 2. If any other kind of blocking (e.g. custom filtering rules, blocklists) works?
 3. Have you got some client-specific settings?

It would be also really helpful if you'd [collect verbose log](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) and send it here or to devteam@adguard.com.  Please try to access some adult site during collecting it.  Thanks

**Comment by StevenKim523** [comment]
Sorry for not providing details!
1. I am using AGH in an openwrt docker container. Also, I'm using ShadowSocksR Plus+ with DNS resolution pointing to AGH.
2. Custom filtering rules work well.
3. There are no client-specific settings
Here is the log
[AdGuardHome1-19-23_42.log](https://github.com/AdguardTeam/AdGuardHome/files/7898436/AdGuardHome1-19-23_42.log)
.

**Comment by EugeneOne1** [comment]
@StevenKim523, it's okay.  Judging by logs, AGH at least tries to block the `pornhub.com`.  You may confirm it by searching the query log for the domain and seeing **Blocked by Parental Control** status for those.

I suppose there is another resolver in your environment or perhaps OpenWRT still uses some additional upstream servers.  Could you please perform a lookup with `dig` or [`dnslookup`](https://github.com/ameshkov/dnslookup).  Those should print the DNS server used.

**Comment by StevenKim523** [comment]
Thanks for your patience!
I found a new issue: When i use AGH only, after i turned on the Parental Control, i can not access any websites. As soon as i turn off the Parental Control, the network restored. It seems that this domain 'dns-family.adguard.com' is blocked by the GFW (By the way, l live in China mainland).

- Here is the log:
[AdGuardHome1-20-15_43.log](https://github.com/AdguardTeam/AdGuardHome/files/7903191/AdGuardHome1-20-15_43.log)

**Comment by EugeneOne1** [comment]
@StevenKim523, according to the log, it's indeed truth.  The situation with safe browsing and parental control services is described in [this comment](https://github.com/AdguardTeam/AdGuardHome/issues/4121#issuecomment-1017636880).  However for now it seems impossible to use the service if it's unavailable from your location.

**Comment by StevenKim523** [comment]
Thanks for your help!😄
